.gitignore

```diff
@@ -1,4 +1,8 @@
 .idea
 .tox
-python-hostsresolver.iml
-MANIFEST
+.eggs
+*.egg-info
+*.iml
```

Comment:
maybe this should go in global ignore so it's not part of the upstream .git file?